CHANGES.rst

```diff
@@ -21,6 +21,7 @@ Improvements
   thanks :user:`Moliholy, unconventionaldotdev`)
 - A room's bookable hours can now be applied to specific weekdays, making it
   unbookable on any other weekdays (:pr:`6439`)
+- Expose registration form image data for use in the check-in app (:pr:`6506`)
```

Comment:
I don't think this needs a changelog entry (but if you want to add one, put it under "Internal Changes"), since we don't maintain a changelog for the checkin app here.